extension/js/common/browser/env.ts

```diff
@@ -41,7 +41,12 @@ export class Env {
   }
 
   // Check if the current context is a Service Worker
-  public static isBackgroundPage() {
+  public static async isBackgroundPage() {
+    // In firefox, window&document is not null even in background page
+    if (typeof browser !== 'undefined' && typeof browser.runtime?.getBackgroundPage !== 'undefined') {
```

Comment:
this check probably can be simplified to just `if (browser?.runtime?.getBackgroundPage !== undefined)`

Reply:
It would throw `Uncaught ReferenceError: browser is not defined` in chrome.

Reply:
Ah, I see, then all good 👍